aprotos/08_CSS3/css081.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
where is media here? task is not finished

Reply:
media in css file